code/game/objects/items/airbubble.dm

```diff
@@ -85,6 +89,10 @@
 		to_chat(user, "<span class='notice'>\The [src] has [internal_tank] attached, that displays [round(internal_tank.air_contents.return_pressure() ? internal_tank.air_contents.return_pressure() : 0)] KPa.</span>")
 	else
 		to_chat(user, "<span class='notice'>\The [src] has no tank attached.</span>")
+	if (cell)
+		to_chat(user, "\The [src] has [cell] attached, the charge meter reads [round(cell.percent())]%.")
+	else
+		to_chat(user, "<span class='warning'>[src] has no power cell installed.</span>")
```

Comment:
Should be "/The [src] has no power cell installed.

Reply:
Unnecessary: the newer (at this point, by years) standard has names properly prepended with `\proper` or `\improper`, which will be utilized when they're put into a string directly. `\the` is required when you use something like `[src.name]`.